ccpp/config/ccpp_prebuild_config.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is this blank line added intentionally?

Reply:
Wow! No, that's quite a bug; the kernel would have used sh to execute that script. Thank you for catching it. I have removed the blank line.